src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+
+    @Override
+    public List<FruitTransaction> convertListToObjects(List<String> list) {
+        List<FruitTransaction> listOfTransactions = new ArrayList<>();
+        for (int i = 1; i < list.size(); i++) {
```

Comment:
Starting the loop at index 1 implies that there is an assumption about the structure of the input list, possibly that the first line is a header. This should be documented in a comment, or better yet, handled more explicitly by checking if the first line is a header.